src/main/java/pl/wojciechkostecki/revel/model/dto/LocalDTO.java

```diff
@@ -0,0 +1,19 @@
+package pl.wojciechkostecki.revel.model.dto;
+
+import lombok.Data;
+import pl.wojciechkostecki.revel.model.Menu;
+
+import java.time.LocalTime;
+
+@Data
+public class LocalDTO {
```

Comment:
Exactly the same just like the Entity

Reply:
I've corrected all DTOs by removing id from them because it is generated automatically.

Reply:
You should keep the id in case we would like to return the DTO, which I would recommend.